.packcheck.ignore

```diff
@@ -0,0 +1,7 @@
+.packcheck.ignore
+.github/workflows/haskell.yml
+.gitignore
+.hlint.yaml
+appveyor.yml
+cabal.project.Werror
+stack.yaml
```

Comment:
We generally add these to the dist?
I remember asking this question before.

Reply:
Should we make it a habit to not add the CI files and hlint and stack files to the dist?

Reply:
A distribution is everything that is required to compile and use the software. If there were a concept of "dev" distribution, we could package all the development related artifacts in that. But the git repo is available for that purpose.